all/pom.xml

```diff
@@ -0,0 +1,121 @@
+<!--
```

Comment:
let's rename `all` as `bookkeeper-all` or `bookkeeper-dist` (as what Leigh suggested in apache/distributedlog#175

Reply:
Thanks, will do it.

Reply:
Thanks. will change it..